seqr/views/apis/staff_api_tests.py

```diff
@@ -260,6 +271,105 @@
      u'animal_model': u'N', u'non_human_cell_culture_model': u'N', u'expression': u'N',
      u'gene_name': u'RP11-206L10.5', u'breast': u'N'}
 
+AIRTABLE_SAMPLE_RECORDS = {
+    "records": [
+        {
+            "id": "rec2B6OGmQpAkQW3s",
+            "fields": {
+                "SeqrCollaboratorSampleID": "19F-DR-1",
+                "CollaboratorSampleID": "VCGS_FAM203_621_D1",
+                "Collaborator": ["recW24C2CJW5lT64K"],
+                "dbgap_study_id": "dbgap_stady_id_1",
+                "dbgap_subject_id": "dbgap_subject_id_1",
+                "dbgap_sample_id": "SM-A4GQ4",
+                "SequencingProduct": [
+                    "Mendelian Rare Disease Exome"
+                ],
+                "dbgap_submission": [
+                    "WES",
+                    "Array"
+                ]
+            },
+            "createdTime": "2019-09-09T19:21:12.000Z"
+        },
+        {
+            "id": "rec2Nkg10N1KssPc3",
+            "fields": {
+                "SeqrCollaboratorSampleID": "19F-DR-2",
+                "CollaboratorSampleID": "VCGS_FAM203_621_D2",
+                "Collaborator": ["reca4hcBnbA2cnZf9"],
+                "dbgap_study_id": "dbgap_stady_id_2",
+                "dbgap_subject_id": "dbgap_subject_id_2",
+                "dbgap_sample_id": "SM-JDBTT",
+                "SequencingProduct": [
+                    "Standard Germline Exome v6 Plus GSA Array"
+                ],
+                "dbgap_submission": [
+                    "WES",
+                    "Array"
+                ]
+            },
+            "createdTime": "2019-07-16T18:23:21.000Z"
+        }
+    ]}
+
+AIRTABLE_COLLABORATOR_RECORDS = {
+    "records": [
+        {
+            "id": "recW24C2CJW5lT64K",
+            "fields": {
+                "CollaboratorID": "Hildebrandt",
+            }
+        },
+        {
+            "id": "reca4hcBnbA2cnZf9",
+            "fields": {
+                "CollaboratorID": "Seidman",
+            }
+        }
+    ]
+}
+
+EXPECTED_PI_SUBJECT_FILE = [
+    u'1kg project n\xe5me with uni\xe7\xf8de_PI_Subject',
+    ['entity:subject_id', 'subject_id', 'prior_testing', 'project_id', 'pmid_id',
+     'dbgap_submission', 'dbgap_study_id', 'dbgap_subject_id', 'multiple_datasets', 'sex',
+     'ancestry', 'ancestry_detail', 'age_at_last_observation', 'phenotype_group', 'disease_id',
+     'disease_description', 'affected_status', 'onset_category', 'age_of_onset', 'hpo_present',
+     'hpo_absent', 'phenotype_description', 'solve_state']
+]
```

Comment:
You need to test against a row that has the airtable data, so pick a row to use that has dbgap_subject_id set

Reply:
As above.